.gitignore

```diff
@@ -31,3 +31,7 @@ bwapi/libReplayToolTest/TestData/Replay_Tool_Actual.exe
 bwapi/ipch/
 Release_Binary/Chaoslauncher/__log.txt
 Release_Binary/ExampleAIClient/Source/
+
+/bwapi/packages/*
+!/bwapi/packages/repositories.config

```

Comment:
Okay, I rebased even though there was a conflict in .gitignore, but it seemed silly making an ugly merge over a one line difference. I just chose your version of it.